src/System.Private.ServiceModel/src/System/ServiceModel/Channels/RTSocketConnection.cs

```diff
@@ -258,6 +259,10 @@ protected override void ShutdownCore(TimeSpan timeout)
                     throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(exceptionToThrow);
                 }
             }
+            catch (OperationCanceledException)
+            {
+                throw;
```

Comment:
Question: why is this new catch and rethrow necessary?  Bug fix or change in the code paths allowed by other changes in this PR? Should it AbortWrite()?  Not blocking -- just questions.